test/Tests.Infrastructure/CommonTestsAssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Xunit;
+
+#if DEBUG || VALIDATE || TEST_PERFORMANCE
+
+[assembly: TestCaseOrderer("Tests.Infrastructure.AlphabeticTestsOrderer", "Tests.Infrastructure")]
+[assembly: TestCollectionOrderer("Tests.Infrastructure.AlphabeticTestsOrderer", "Tests.Infrastructure")]
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Let's control that via xunit.setting.json

Reply:
I have removed the DisableTestParallelization, as we discussed.